examples/gno.land/p/demo/acl/acl.gno

```diff
@@ -56,47 +56,229 @@ func (d *Directory) HasRole(addr std.Address, role string) bool {
 	return d.HasPerm(addr, "role", role)
 }
 
-func (d *Directory) AddUserPerm(addr std.Address, verb, resource string) {
+func (d *Directory) AddUserPerm(addr std.Address, verb, resource string) bool {
 	bucket := "u:" + addr.String()
 	p := perm{
 		verbs:     []string{verb},
 		resources: []string{resource},
 	}
-	d.addPermToBucket(bucket, p)
+	d.addPermsToBucket(bucket, []perm{p})
+	return true
 }
 
-func (d *Directory) AddGroupPerm(name string, verb, resource string) {
+func (d *Directory) AddUserPerms(addr std.Address, verbs []string, resource string) bool {
```

Comment:
this helper should take multiple verbs AND resources, or multiple structs, not just multiple verbs.